### ISSUE TITLE
Is postgresql.CIDR can be added to the convert mapping ?

### ISSUE THREAD

**Issue by chaufmaurice** [marshmallow-code/marshmallow-sqlalchemy#183]
Hello,

My
```
subnet = field_for(Layer3, 'subnet', example='10.123.50.0/30')
```
generate a
```
marshmallow_sqlalchemy.exceptions.ModelConversionError: Could not find field column of type <class 'sqlalchemy.dialects.postgresql.base.CIDR'>.
```

The database column concerned:
```
subnet CIDR NOT NULL,
```

It looks like CIDR is not include in the convert mapping:
https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/637077fad43830f035c84b7e9969cb4c84f4267f/marshmallow_sqlalchemy/convert.py#L42

Is there any reason ?
Thank you.

**Comment by sloria** [comment]
The omission was not intentional. Feel free to send a PR!

**Comment by sloria** [comment]
Fixed in 0.20.0
